CASMTRIAGE-5937 Update sls.md with Authenication Token details. (#4178)
Cray-HPE/docs-csm: api/sls.md

Endpoints which request information about hardware

> Refer link [Retrieve an Authentication Token](Retrieve_an_Authentication_Token.md) to get Authentication Token before using the code samples.

Reviewer: ```suggestion
Refer to the ['Retrieve an Authentication Token'](Retrieve_an_Authentication_Token.md) documentation for information regarding retrieving an authentication token.
```

```

```shell

> 

Reviewer 2: ```suggestion
```
